food/parallel.js

```diff
@@ -123,13 +122,6 @@ d3.csv("nutrients.csv", function(raw_data) {
           g.attr("transform", function(d) { return "translate(" + position(d) + ")"; });
           brush_count++;
           this.__dragged__ = true;
-
-          // Feedback for axis deletion if dropped
```

Comment:
is this removing the ability to remove an axis?

Reply:
yes. This block specifically is handling the style (the red rect), and a block somewhere else is responsible for removing it.

Reply:
red rectangle?